hack/tools/release/notes.go

```diff
@@ -73,6 +73,8 @@ var (
 
 	prefixAreaLabel = flag.Bool("prefix-area-label", true, "If enabled, will prefix the area label.")
```

Comment:
No idea why but in some cases apparently the flags package already adds the default in other cases it doesn't?

```bash
 go run ./hack/tools/release/notes.go --help
Usage of /var/folders/v0/mny6w2dd33gfzwmwxb3wyyx40000gq/T/go-build4290625525/b001/exe/notes:
  -add-kubernetes-version-support
        If enabled, will add the Kubernetes version support header. (default true)
  -deprecation
        If enabled, will add a templated deprecation warning header. (default true) (default true)
  -from string
        The tag or commit to start from.
  -pre-release-version
        If enabled, will add a pre-release warning header. (default false)
  -prefix-area-label
        If enabled, will prefix the area label. (default true)
  -repository string
        The repo to run the tool from. (default "kubernetes-sigs/cluster-api")
  -since string
        Include commits starting from and including this date. Accepts format: YYYY-MM-DD
  -until string
        Include commits up to and including this date. Accepts format: YYYY-MM-DD
  -workers int
        Number of concurrent routines to process PR entries. If running into GitHub rate limiting, use 1. (default 10)

```